app/javascript/packs/red_hat_cloud_services_list.jsx

```diff
@@ -136,7 +147,7 @@ const RedHatCloudServicesList = () => {
               </Filter>
             </div>
             <div class="form-group">
-              <button class="btn btn-default" type="button" id="Synchronize" disabled={rows.filter(row => row.selected == true).length == 0}>{__('Synchronize')}</button>
+              <button class="btn btn-default" type="button" id="Synchronize" disabled={rows.filter(row => row.selected == true).length == 0} onClick={() => SyncProviders(rows.filter(row => row.selected == true))}>{__('Synchronize')}</button>
```

Comment:
@Fryguy @abellotti we are missing onclick here https://github.com/RedHatCloudForms/cfme-cloud_services/blob/7cf11f993f2f336da9bb5ef5eb8ce38ab3849043/app/javascript/packs/red_hat_cloud_services_list.jsx#L118

Reply:
That's the button to Sync all of MIQ data to cloud.

Reply:
Thanks @h-kataria for 👀 this over.

While we can leverage the bulk action (as used by the SyncProviders method) and specify id's of all providers, it may be better to support a sync_all action on the collection (no need for an initial API call to fetch all id's).  Then we can wire that button to it.  Lemme think about this a bit more.

Reply:
Ok, sync_all will need a minor enhancement to the base API to support collection level actions, probably declared as such in the api.yml,  will use the bulk action for now.

Reply:
> Thanks @h-kataria for eyes this over.
> 
> While we can leverage the bulk action (as used by the SyncProviders method) and specify id's of all providers, it may be better to support a sync_all action on the collection (no need for an initial API call to fetch all id's). Then we can wire that button to it. Lemme think about this a bit more.

You are right we should not need to fetch id's of all providers on initial call, as far as i know Sync Platform button not only uploads all providers data but other MIQ objects as well(i.e Catalog Items etc.). @Fryguy please correct me if I am wrong.

Reply:
That is correct...we also need the "core" collection.